### PR TITLE
feature/change-volume-component

### PR DESCRIPTION
**Pull request by JaHajRF**
volume bar changed to fit new design and set volume in store

### CODE REVIEW

File: packages/webapp/src/components/NavBar/NavBar.types.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
+import { WebPlayerActions } from '../../store/webPlayer/webPlayer.reducer'
+
 export interface INavBarProps {
-  changeSongVolume: (value: number) => void
   linkUrl: string
   onSocialShare: () => void
   roomName: string
+  setWebPlayerVolume: (volume: number) => WebPlayerActions['setVolume']
```
Comment (kowalski) [inline packages/webapp/src/components/NavBar/NavBar.types.ts:7]: is this right typing ? 
imo it should be 
```
(volume: number) => void
```
no?

File: packages/webapp/src/store/webPlayer/webPlayer.saga.ts [hunk 2]
```diff
@@ -197,12 +197,22 @@ export function* handleSongPreviewSong(action: WebPlayerActions['playSongPreview
   }
 }
 
+export function* setWebPlayerVolume(action: WebPlayerActions['setVolume']): Generator {
+  const playedSong = (yield select(webPlayerSelectors.playedSong)) as HTMLAudioElement | null
```
Comment (kowalski) [inline packages/webapp/src/store/webPlayer/webPlayer.saga.ts:201]: btw, these `as HTMLAudioElement | null` are bugging me

Can we add typed-redux-saga ?

**Review by kowalski — CHANGES_REQUESTED** [review]

File: packages/webapp/src/components/NavBar/NavBar.types.ts (hunk 1 above)
Comment (JaHajRF) [reply packages/webapp/src/components/NavBar/NavBar.types.ts:7]: I used to set it as void but I've changed that here. My reasoning:

In container it is typed as: 
`const setWebPlayerVolume = (volume: number): WebPlayerActions['setVolume'] =>
    dispatch(webPlayerActions.setVolume(volume))`

So I've followed useDispatch typing and: 
`export interface Dispatch<A extends Action = AnyAction> {
  <T extends A>(action: T): T
}`

as dispatch returns action object I assumed that would be proper typing.

Both ways are valid for compiler.
What do you think?

File: packages/webapp/src/store/webPlayer/webPlayer.saga.ts (hunk 2 above)
Comment (JaHajRF) [reply packages/webapp/src/store/webPlayer/webPlayer.saga.ts:201]: Ok I will make github issue and work on that

**Review by kowalski — APPROVED** [review]

**Pull request merged**